config/flags_service_shared.go

```diff
@@ -68,18 +61,25 @@ var (
 		Usage: `Proposal fetch interval { "30s", "3m", "1h20m30s" }`,
 		Value: 10 * time.Minute,
 	}
+
+	// FlagPaymentPricePerGB sets the price per GiB to provided service.
+	FlagPaymentPricePerGB = cli.Float64Flag{
```

Comment:
Why is this moved to shared if each service should be able to set it's own price?

Reply:
Implemented per service parameters for prices and access policies also